e2e/rbd.go

```diff
@@ -370,6 +370,45 @@ var _ = Describe("RBD", func() {
 				})
 			}
 
+			By("Validate AnnotateFSResize", func() {
+				// this featuregate is available since kube v1.21.
+				expandSize := "3Gi"
+				pvc, err := loadPVC(pvcPath)
+				if err != nil {
+					e2elog.Failf("failed to load PVC: %v", err)
+				}
+				pvc.Namespace = f.UniqueName
+
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create PVC: %v", err)
+				}
+				// validate created backend rbd images
+				validateRBDImageCount(f, 1, defaultRBDPool)
+
+				// resize PVC
+				err = expandPVCSize(f.ClientSet, pvc, expandSize, 1)
```

Comment:
if the expansion is successful, this test should fail, or else the below case is not verified.

Reply:
true.. the only aim of this test is to prove the annotation getting popped up ..